src/api/rasters.ts

```diff
@@ -54,68 +54,69 @@ export const fetchRasterV4 = async (uuid: string, options: RequestInit = {
 };
 
 export const flushRaster = async (uuid: string) => {
+  return Promise.resolve();
```

Comment:
I suppose below commented out codes are for later use? flushRaster is not working right now.